automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingApiSpec.scala

```diff
@@ -32,99 +35,115 @@ class BillingApiSpec extends FreeSpec with BillingFixtures with MethodFixtures w
 
-        // clean up
-        Rawls.workspaces.delete(billingProjectName, workspaceName)
-        deleteBillingProject(billingProjectName)
+      // add studentA to google billing project with USER role
+      val studentA = UserPool.chooseStudent
+      Rawls.billing.addUserToBillingProject(billingProjectName, owner.email, BillingProjectRole.Owner)
+      Rawls.billing.addUserToBillingProject(billingProjectName, studentA.email, BillingProjectRole.User)
+
+      // verify studentA is a member in google billing project
+      val members: List[Map[String, String]] = Rawls.billing.listMembersInBillingProject(billingProjectName)
+      val emails: List[String] = members.map(_.getOrElse("email", ""))
+      emails should (contain(studentA.email) and not contain "")
+
+      // create new workspace
+      val workspaceName = randomIdWithPrefix("BillingApiSpec")
+      register cleanUp Try(Rawls.workspaces.delete(billingProjectName, workspaceName)).recover {
+        case _: RestException =>
+      }
+      Rawls.workspaces.create(billingProjectName, workspaceName)
+      Orchestration.workspaces.waitForBucketReadAccess(billingProjectName, workspaceName)
+
+      // create a method
+      withMethod("BillingApiSpec_workspace", MethodData.SimpleMethod) { methodName =>
+        val method: Method = MethodData.SimpleMethod.copy(methodName = methodName)
+
+        val participantId = randomIdWithPrefix("participant")
+        val participantEntity = s"entity:participant_id\n$participantId"
+        Orchestration.importMetaData(billingProjectName, workspaceName, "entities", participantEntity)
+
+        // create method config in workspace
+        Rawls.methodConfigs.createMethodConfigInWorkspace(
+          billingProjectName,
+          workspaceName,
+          method,
+          SimpleMethodConfig.configNamespace,
+          SimpleMethodConfig.configName,
+          1,
+          SimpleMethodConfig.inputs,
+          SimpleMethodConfig.outputs,
+          SimpleMethodConfig.rootEntityType)
+
+        // launch submission
+        val submissionId = Rawls.submissions.launchWorkflow(
+          billingProjectName,
+          workspaceName,
+          SimpleMethodConfig.configNamespace,
+          SimpleMethodConfig.configName,
+          SimpleMethodConfig.rootEntityType,
+          participantId,
+          "this",
+          useCallCache = false)
+
+        // wait until submission complete
+        Submission.waitUntilSubmissionComplete(billingProjectName, workspaceName, submissionId)
+
+        // verify submission status is Done
+        val expectedStatus = "Done"
+        val actualStatus = Submission.getSubmissionStatus(billingProjectName, workspaceName, submissionId)
+        withClue(s"Submission $billingProjectName/$workspaceName/$submissionId status should be $expectedStatus") {
+          actualStatus shouldBe expectedStatus
+        }
       }
-    }
 
+      // clean up
+      Rawls.workspaces.delete(billingProjectName, workspaceName)
```

Comment:
No longer checking if the billing project can be deleted?

Reply:
Fixed. 👍